src/scripts/ci_build.py

```diff
@@ -257,6 +258,12 @@ def sanitize_kv(some_string):
     if target in ['coverage', 'valgrind', 'valgrind-full', 'valgrind-ct', 'valgrind-ct-full']:
         flags += ['--with-debug-info']
 
+    if target in ['strubbing']:
+        # Stack scrubbing tests are based on scripted gdb runs and won't work on
+        # an optimized build unfortunately.
+        flags += ['--debug-mode']
```

Comment:
`--debug-mode` is just a shorthand for `--no-optimizations` + `--with-debug-info`, what's happening might be more clear using those directly